src/windows/wintab.c

```diff
@@ -335,5 +344,7 @@ bool wintab_on_proximity(struct wintab *ctx, MTY_Event *evt, LPARAM lparam)
 
 void wintab_on_infochange(struct wintab **ctx, HWND hwnd)
 {
-	wintab_recreate(ctx, hwnd, ctx && *ctx && (*ctx)->override);
+	bool override = ctx && *ctx && (*ctx)->override;
+	wintab_destroy(ctx, false);
+	*ctx = wintab_create(hwnd, override);
```

Comment:
```suggestion
	if (!ctx)
		return;

	bool override = *ctx && (*ctx)->override;
	wintab_destroy(ctx, false);
	*ctx = wintab_create(hwnd, override);
```
only the boolean assignment was checking if ctx was a null pointer, letting wintab_create possibly assign to the null pointer and cause an access violation.